impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -329,6 +358,49 @@ public ClientBuilder setOrgUrl(String baseUrl) {
         return this;
     }
 
+    @Override
+    public ClientBuilder setAuthorizationMode(String authorizationMode) {
+        if (Strings.isEmpty(authorizationMode)) {
+            throw new IllegalArgumentException("Invalid AuthorizationMode");
+        }
+        this.clientConfig.setAuthorizationMode(authorizationMode);
+        return this;
+    }
+
+    @Override
+    public ClientBuilder setScopes(List<String> scopes) {
+        if (scopes == null || scopes.size() == 0) {
+            throw new IllegalArgumentException("Invalid Scopes");
+        }
+        this.clientConfig.setScopes(scopes);
+        return this;
+    }
+
+    @Override
+    public ClientBuilder setKeyFilePath(String keyFilePath) {
+        if (Strings.isEmpty(keyFilePath)) {
+            throw new IllegalArgumentException("Invalid KeyFilePath");
```

Comment:
Take a look at the `Assert` class, it can do this do this in a single line

Reply:
noted. will use.